### DISCUSSION TITLE
Not getting password email after creating new user in admin panel

### DISCUSSION THREAD

**Discussion by wilfing-dominik** [chiiya/filament-access-control#52] [Q&A] [answered]
I followed the readme but I don't seem to get any kind of email after creating a new user in filament. Did I miss something?

**Comment by fahmiegerton** [comment]
Same thing happened to me. How can I set password to myself instead generated one?

**Reply by chiiya** [reply]
It's probably your mail configuration rather than the package, but I'll have a look. You can try resetting your password on the login page. Otherwise something is definitely wrong with your mail configuration.

**Reply by fahmiegerton** [reply]
I did, and I tried with another laravel app, it's working, but I will investigate it more

**Reply by fahmiegerton** [reply, marked as answer]
Turns out, it need queue:work to run, how dumb I am, thank you!
